app/views/users/memes.html.erb

```diff
@@ -0,0 +1,26 @@
+<% provide(:title, "My Memes") %>
+
+<div class="row">

```

Comment:
This common div/ul/li stuff is the same on all 3 views—just put it in a common file that you use the <% yield %> with?